tst/EnergyPlus/unit/SolarShading.unit.cc

```diff
@@ -6250,8 +6250,8 @@ TEST_F(EnergyPlusFixture, CLIPLINE_Full)
         Line line_new{}; // Only defined if visible
     };
 
-    // No need to capture the constexpr variables
-    auto testclipline = [](const TestCase &t) {
+    // No need to capture the constexpr variables (but MSVC will complain)
+    auto testclipline = [&](const TestCase &t) {
```

Comment:
```
D:\a\EnergyPlus\EnergyPlus\tst\EnergyPlus\unit\SolarShading.unit.cc(6265): error C3493: 'maxX' cannot be implicitly captured because no default capture mode has been specified
```

Reply:
Absurd.  😆